ocaml/idl/datamodel.ml

```diff
@@ -9019,7 +9019,7 @@ let pgpu =
         {
           param_type = (Ref _vgpu_type);
           param_name = "value";
-          param_doc = "The VGPU type to enable";
+          param_doc = "The VGPU types to enable";
```

Comment:
No, that one is correct in plural, because it's a `Set (Ref _vgpu_type)` (many of them).